migrations/20200708164116_add_ra_date_to_projects.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex) {
+  return knex.schema.table('projects', table => {
+    table.dateTime('ra_date').nullable();
```

Comment:
```suggestion
    table.date('ra_date').nullable();
```

We don't need time precision for this.